packages/strapi-helper-plugin/lib/src/components/Input/index.js

```diff
@@ -139,8 +139,15 @@ class Input extends React.Component { // eslint-disable-line react/prefer-statel
           {title}
           <FormattedMessage id={this.props.label} defaultMessage={this.props.label}>
             {(message) => (
-              <label className={`${styles.checkboxLabel} form-check-label`} htmlFor={this.props.label}>
-                <input className="form-check-input" type="checkbox" defaultChecked={this.props.value} onChange={this.handleChangeCheckbox} name={this.props.name} />
+              <label className={`${styles.checkboxLabel} form-check-label`} htmlFor={this.props.name} style={{ cursor: 'pointer' }}>
```

Comment:
There is a `checkboxLabel` applied to this block, why are you passing style with the `style` attribute.